pkg/uninstall/uninstall.go

```
@@ -115,6 +119,24 @@ This will uninstall Everest and all monitoring resources deployed by it. All oth
 	return nil
 }
 
+func (u *Uninstall) areResourcesExist(ctx context.Context) error {
+	_, err := u.kubeClient.GetNamespace(ctx, u.config.Namespace)
+	if err != nil && k8serrors.IsNotFound(err) {
+		return fmt.Errorf("namespace %s is not found", u.config.Namespace)
```

Why do we throw an error if we check if resources exists?
They don't exist, which is fine, right?

---

We need to throw an error and break the process according to the ticket description. I just made it clear to the user